docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,375 @@
+
+<!-- Piotr Jamroz -->
+
+_Available in public preview in all editions of Grafana._
+
+You can now query multiple data sources simultaneously in Explore. Select "Mixed" from the data source picker and specify a data source for each query.
+
+Mixed data source in Explore is gradually rolling out to all users on Grafana Cloud. If you’re using Grafana Open Source and Enterprise, you can enable this feature using the `exploreMixedDatasource` feature toggle.
+
+## Public dashboards
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
+
+Public dashboards allow you to share your Grafana dashboard with anyone without requiring them to log in to Grafana. This is useful when you want to make your dashboard available to the world.
+
+With this update, we've made the following improvements:
+
+- The time picker and annotations can be toggled on or off in public dashboard configuration
+- You can see a list of all your public dashboards in **Dashboards > Public dashboards**
+- Improved UI: new modal design, paused and not found pages
+- Added support for collapsed rows, hidden queries, and zoom into panels
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+To learn more, check out our public dashboards [documentation]({{< relref "../../dashboards/dashboard-public" >}}).
+
+### Public dashboards insights
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+Public dashboards insights provide valuable information about your public dashboard usage. You can easily access and view important metrics such as the daily query count, the number of views in the last 30 days, and the number of errors in the last 30 days.
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+### Email sharing for public dashboards
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can view your public dashboards.
+
+We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.
+
+To try it out, you'll need to enable the `publicDashboardsEmailSharing` and `publicDashboards` feature toggles. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```

Comment:
```suggestion
_Available in preview in Cloud Pro, and Cloud Advanced._

Our email sharing feature allows you to easily share your public dashboards and make them visible only with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can view your public dashboards.

We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.

To try it out, please contact customer support.

This feature will have a cost by active users after being promoted into general availability.
```

Reply:
@imatwawana please help me improve these changes:
 * Changed versions, now only available on Cloud
 * Removed any mentions of feature flags
 * Added a comment to state that only users that received an invitation will have access to the public dashboard as @timlevett suggested.
 * Explained this feature will have a cost
 
cc: @thanos-karachalios @natellium

Reply:
I think this is fine; I've made one small wording tweak and added "private" to the availability note, to denote that this feature is only available to invited customers.

Reply:
I think this fits better with the concept of `cloud public preview` from our new [Release life cycle](https://docs.google.com/document/d/11SD-DSBtHDyXOzPVlq82cwy3XeeBhqClejUorUi86Hw/edit#) because we are announcing it publicly and it will available to anyone who requests access.

Reply:
Ah okay, you're right. I misunderstood.